app/google-services.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
Is it okay to publish this file in the public repo?

Reply:
Hmm... Good question.
Will need to investigate it

Reply:
Though I already made it public by committing this changes.... 🤔

Reply:
Have checked a few questions in StackOverflow regarding this. And the general idea is that it is not a real security breach. But does expose some endpoints. So it is not recommended to have it fully public. One of the replies from Google support:
> The general answer is yes, the google-services.json is safe to check into your repo and is something that should be shared among engineers on your team.  The JSON file does not contain any super-sensitive information (like a server API key).  It does contain some information like your database URL, Android API key, and storage bucket.  These are not secrets, but if your security rules are not set up correctly an attacker could use them against you.  However since these values are also compiled into your APK as resources, the theoretical attacker would not need your JSON file to get these values anyway.

Reply:
I will bring this topic tomorrow on standup

Reply:
Change the way the Firebase is initialised